components/Comment/CommentActions.tsx

```diff
@@ -1,25 +1,92 @@
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { faReply, faEdit } from "@fortawesome/free-solid-svg-icons";
+import { faPencil } from "@fortawesome/pro-light-svg-icons";
 import { css, StyleSheet } from "aphrodite";
+import CommentVote from "./CommentVote";
+import { parseUser, TopLevelDocument } from "~/config/types/root_types";
+import { Comment } from "./lib/types";
+import Image from "next/image";
+import IconButton from "../Icons/IconButton";
+import colors from "./lib/colors";
+import { isEmpty } from "~/config/utils/nullchecks";
+import { useSelector } from "react-redux";
+import { RootState } from "~/redux";
 
 type Args = {
   handleEdit: Function;
   handleReply: Function;
+  comment: Comment;
+  document: TopLevelDocument;
 };
 
-const CommentActions = ({ handleEdit, handleReply }: Args) => {
+const CommentActions = ({
+  comment,
+  document,
+  handleEdit,
+  handleReply,
+}: Args) => {
+  const currentUser = useSelector((state: RootState) =>
+    isEmpty(state.auth?.user) ? null : parseUser(state.auth.user)
+  );
+
   return (
     <div className={css(styles.wrapper)}>
       <div className={css(styles.actionsWrapper)}>
         <div className={`${css(styles.action)} reply-btn`}>
-          {/* TODO: This requires updating font awesome common types */}
-          <FontAwesomeIcon icon={faReply} />
-          <span onClick={() => handleReply()}>Reply</span>
+          <CommentVote
+            comment={comment}
+            score={comment.score}
+            userVote={comment.userVote}
+            documentType={document.documentType}
+            documentID={document.id}
+          />
+        </div>
+        <div className={`${css(styles.action)} reply-btn`}>
+          <IconButton onClick={() => null}>
+            <Image
+              src="/static/icons/tip.png"
+              height={20}
+              width={21}
+              alt="Tip"
+            />
+            <span
+              className={css(styles.actionText)}
+              onClick={() => handleReply()}
+            >
+              Tip
+            </span>
+          </IconButton>
         </div>
-        <div className={`${css(styles.action)} edit-btn`}>
-          {/* TODO: This requires updating font awesome common types */}
-          <FontAwesomeIcon icon={faEdit} />
-          <span onClick={() => handleEdit()}>Edit</span>
+        {currentUser?.id === comment.createdBy.id && (
+          <div className={`${css(styles.action, styles.editAction)} edit-btn`}>
+            <IconButton onClick={() => null}>
+              <FontAwesomeIcon
+                icon={faPencil}
+                style={{ color: colors.secondary.text, fontSize: 18 }}
+              />
+              <span
+                className={css(styles.actionText)}
+                onClick={() => handleEdit()}
```

Comment:
Same as above

Reply:
This is is WIP. My next PR will have something in there. Just needed a placeholder now.